tasks/src/tabular-regression/about.md

```diff
@@ -3,7 +3,15 @@
 Tabular regression is the task of predicting a numerical value given a set of attributes/features. *Tabular* meaning that data is stored in a table (like an excel sheet), and each sample is contained in its own row. The features used to predict our target can be both numerical and categorical. However, including categorical features often requires additional preprocessing/feature engineering (a few models do accept categorical features directly, like [CatBoost](https://catboost.ai/)). An example of tabular regression would be predicting the weight of a fish given its' species and length.
 
 ## Use Cases
+Tabular data is one of the most prevalent data out there, and many raw-datasets once we extract features from them can be arrange in tabular form. Below you can find some of the popular tabular use-cases:-
```

Comment:
```suggestion
Tabular data is the most available data in the industry. Many raw datasets, once we extract features from them, can be arranged in tabular form. Below you can find some of the popular tabular regression use cases:
```

Reply:
Yeah no worries, I am really thankful for your detailed feedback. I want to develop my technical writing skill to communicate ideas as simply as possible!